### ISSUE TITLE
Duplicates users

### ISSUE THREAD

**Issue by damianooldoni** [inbo/sovon#136]
From e-mail of @JeroenNienhuis:

> Een aantal waarnemers staan dubbel in de aangeleverde data
109 --> 110
170 --> 481
194 --> 178
2088 --> 603
552 --> 350
699 --> 571
244 --> 78

BY a first fast check, this is indeed quite true. Sometimes are first name and last name inverted, sometimes is email filled in but no first/last name provided and viceversa.

@EricStienen: could you please double check this? In any case I suggest to solve this at crbirding database level to avoid modifying data in our db as we decided to not add or change anything anymore in it. Is it ok for you, @JeroenNienhuis, to solve this while inserting data in SOVON's db?

**Comment by EricStienen** [comment]
Yes these are doubles indeed. If this can be solved at the level of the SOVON db, I would be happy.

**Comment by JeroenNienhuis** [comment]
I will solve this when inserting the data. If there are more double users please let me know.
